js/battle.js

```diff
@@ -5464,6 +5464,7 @@ var Battle = (function () {
 			this.log('<div class="debug"><div class="chat"><small style="color:#999">[DEBUG] ' + Tools.escapeHTML(name) + '.</small></div></div>', preempt);
 			break;
 		case 'unlink':
+			if (Tools.prefs('nounlinkhide')) return;
```

Comment:
You use `nohideunlink` in the commands and `nounlinkhide` here.

If you're having trouble remembering which it is, this usually means that the pref is too long. How about `nounlink`?